app/assets/javascripts/lesson_planner/lesson_planner.jsx

```diff
@@ -188,6 +188,15 @@ EC.LessonPlanner = React.createClass({
     });
   },
 
+	// TODO: remove staging from the URL or build a regex that grabs everythig before the third slash using window.location.href
+	getInviteStudentsUrl: function() {
+		return ('https://staging.quill.org/teachers/classrooms/' + $(".tab-pane").data().classroomId + '/invite_students');
```

Comment:
This should just be /teachers/classrooms etc so that it works in any environment